proto/wechaty/puppet/friendship.proto

```diff
@@ -62,7 +62,7 @@ message FriendshipSearchWeixinResponse {
 
 message FriendshipAddRequest {
   string contact_id = 1;
-  string hello = 2;
+  string options = 2;
```

Comment:
This is not acceptable.

In gRPC, the field should neither be renamed not be reused.

Please keep the original `hello`, and we can use `options = 3;` instead.